uiplib/scrape.py

```diff
@@ -36,23 +36,27 @@ def get_images(url):
     '''
     scrape images from /r/wallpapers
     '''
-    soup = make_soup(url)
-    # this makes a list of bs4 element tags
-    thumbnails = soup.find_all("a", class_="thumbnail", href=True)
+    image_links = []
+    p_url = url
+    while len(image_links) < NUMBER_OF_IMAGES_TO_PARSE:
```

Comment:
See the lines 58,59 actually i am navigating through every page of reddit wallpapers  until i get the required number of wallpapers.Even if I dont find the required number of images in the first page I will find it after few number of pages